surface/sca/management/commands/resync_vulns_counters.py

```diff
@@ -6,14 +6,16 @@
 
 
 class Command(LogBaseCommand):
-    help = "Re-sync SCA Projects Vulnerabilities Counters"
+    help = "Re-sync SCA Projects Vulnerabilities Counters and dependencies list."
 
     processed_projects = 0
 
     def handle(self, *args, **options):
         self.sync_time = timezone.now()
         for project in tqdm.tqdm(SCADependency.objects.filter(is_project=True)):
             project.update_vulnerability_counters()
+            project.dependencies_list = project.dependencies
+            project.save()
```

Comment:
```suggestion
            project.save(update_fields=["dependencies_list"])
```

https://docs.djangoproject.com/en/5.2/ref/models/instances/#specifying-which-fields-to-save